subiquity/server/curtin.py

```diff
@@ -132,24 +130,48 @@ async def run(self, context):
 
 class _DryRunCurtinCommand(_CurtinCommand):
 
-    event_file = 'examples/curtin-events.json'
+    stages_mapping = {
+        tuple(): "examples/curtin-events-initial.json",  # no stage
+        ("partitioning",): "examples/curtin-events-partitioning.json",
+        ("extract",): "examples/curtin-events-extract.json",
+        ("curthooks",): "examples/curtin-events-curthooks.json",
+        ("hook",): "examples/curtin-events-hook.json",
+    }
 
     def make_command(self, command, *args, config=None):
         if command == 'install':
+            # Lookup the log file from the config if specified
+            try:
+                with open(config, mode="r") as fh:
+                    log_file = yaml.safe_load(fh)["install"]["log_file"]
+            except (TypeError, AttributeError):
+                log_file = "/dev/stderr"
```

Comment:
Do we expect to hit this case? It feels a bit strange...

Reply:
Only when writing a custom test-case (or unit test)... 
and it shows that this code is unused because I meant to use "/dev/null" instead of "/dev/stderr"..